test/built-ins/Temporal/ZonedDateTime/from/argument-propertybag-calendar-wrong-type.js

```diff
@@ -9,20 +9,23 @@ description: >
 features: [BigInt, Symbol, Temporal]
 ---*/
 
+const timeZone = "UTC";
```

Comment:
I see you found a pre-existing error in this test, we forgot the time zone. Thanks!

(Well, technically it's not an error; in step 4.b-c of [ToTemporalZonedDateTime](https://tc39.es/proposal-temporal/#sec-temporal-totemporalzoneddatetime) the check for a faulty calendar occurs before the check for a missing time zone. So the test was still correct, but it was testing something different than I expected.)

Normally I would suggest that we should make a separate test file to ensure that the "faulty calendar" exception is thrown before the "missing time zone" exception, but in this case I think no further action is needed. We generally don't test that sort of thing anyway currently. (See also #2746 for a draft of how we might test it in the future.)